[2025-01-19T20:07:03.077+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-01-19T20:07:03.087+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: coleta_dados_abertos_cnpj.echo_hello_world scheduled__2024-11-10T00:00:00+00:00 [queued]>
[2025-01-19T20:07:03.092+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: coleta_dados_abertos_cnpj.echo_hello_world scheduled__2024-11-10T00:00:00+00:00 [queued]>
[2025-01-19T20:07:03.093+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 2
[2025-01-19T20:07:03.123+0000] {taskinstance.py:2889} INFO - Executing <Task(BashOperator): echo_hello_world> on 2024-11-10 00:00:00+00:00
[2025-01-19T20:07:03.128+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=379) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-01-19T20:07:03.129+0000] {standard_task_runner.py:72} INFO - Started process 386 to run task
[2025-01-19T20:07:03.129+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'coleta_dados_abertos_cnpj', 'echo_hello_world', 'scheduled__2024-11-10T00:00:00+00:00', '--job-id', '99', '--raw', '--subdir', 'DAGS_FOLDER/coleta_dados_abertos_cnpj/dag.py', '--cfg-path', '/tmp/tmpoau0117a']
[2025-01-19T20:07:03.131+0000] {standard_task_runner.py:105} INFO - Job 99: Subtask echo_hello_world
[2025-01-19T20:07:03.159+0000] {task_command.py:467} INFO - Running <TaskInstance: coleta_dados_abertos_cnpj.echo_hello_world scheduled__2024-11-10T00:00:00+00:00 [running]> on host 659dab735c6e
[2025-01-19T20:07:03.210+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Corvaum' AIRFLOW_CTX_DAG_ID='coleta_dados_abertos_cnpj' AIRFLOW_CTX_TASK_ID='echo_hello_world' AIRFLOW_CTX_EXECUTION_DATE='2024-11-10T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-11-10T00:00:00+00:00'
[2025-01-19T20:07:03.211+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-01-19T20:07:03.220+0000] {subprocess.py:78} INFO - Tmp dir root location: /tmp
[2025-01-19T20:07:03.220+0000] {subprocess.py:88} INFO - Running command: ['/usr/bin/bash', '-c', 'echo "Hello, Airflow!"']
[2025-01-19T20:07:03.227+0000] {subprocess.py:99} INFO - Output:
[2025-01-19T20:07:03.228+0000] {subprocess.py:106} INFO - Hello, Airflow!
[2025-01-19T20:07:03.228+0000] {subprocess.py:110} INFO - Command exited with return code 0
[2025-01-19T20:07:03.242+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-01-19T20:07:03.243+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=coleta_dados_abertos_cnpj, task_id=echo_hello_world, run_id=scheduled__2024-11-10T00:00:00+00:00, execution_date=20241110T000000, start_date=20250119T200703, end_date=20250119T200703
[2025-01-19T20:07:03.263+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-01-19T20:07:03.272+0000] {taskinstance.py:3895} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-01-19T20:07:03.273+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
[2025-01-24T19:03:18.039+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-01-24T19:03:18.047+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: coleta_dados_abertos_cnpj.echo_hello_world scheduled__2024-11-10T00:00:00+00:00 [queued]>
[2025-01-24T19:03:18.052+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: coleta_dados_abertos_cnpj.echo_hello_world scheduled__2024-11-10T00:00:00+00:00 [queued]>
[2025-01-24T19:03:18.052+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 2
[2025-01-24T19:03:18.059+0000] {taskinstance.py:2889} INFO - Executing <Task(BashOperator): echo_hello_world> on 2024-11-10 00:00:00+00:00
[2025-01-24T19:03:18.063+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=1557) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-01-24T19:03:18.064+0000] {standard_task_runner.py:72} INFO - Started process 1560 to run task
[2025-01-24T19:03:18.064+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'coleta_dados_abertos_cnpj', 'echo_hello_world', 'scheduled__2024-11-10T00:00:00+00:00', '--job-id', '655', '--raw', '--subdir', 'DAGS_FOLDER/coleta_dados_abertos_cnpj/dag.py', '--cfg-path', '/tmp/tmpyy3u241h']
[2025-01-24T19:03:18.065+0000] {standard_task_runner.py:105} INFO - Job 655: Subtask echo_hello_world
[2025-01-24T19:03:18.088+0000] {task_command.py:467} INFO - Running <TaskInstance: coleta_dados_abertos_cnpj.echo_hello_world scheduled__2024-11-10T00:00:00+00:00 [running]> on host ee70781c29c7
[2025-01-24T19:03:18.131+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Corvaum' AIRFLOW_CTX_DAG_ID='coleta_dados_abertos_cnpj' AIRFLOW_CTX_TASK_ID='echo_hello_world' AIRFLOW_CTX_EXECUTION_DATE='2024-11-10T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-11-10T00:00:00+00:00'
[2025-01-24T19:03:18.132+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-01-24T19:03:18.139+0000] {subprocess.py:78} INFO - Tmp dir root location: /tmp
[2025-01-24T19:03:18.139+0000] {subprocess.py:88} INFO - Running command: ['/usr/bin/bash', '-c', 'echo "Hello, Airflow!"']
[2025-01-24T19:03:18.144+0000] {subprocess.py:99} INFO - Output:
[2025-01-24T19:03:18.144+0000] {subprocess.py:106} INFO - Hello, Airflow!
[2025-01-24T19:03:18.145+0000] {subprocess.py:110} INFO - Command exited with return code 0
[2025-01-24T19:03:18.157+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-01-24T19:03:18.157+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=coleta_dados_abertos_cnpj, task_id=echo_hello_world, run_id=scheduled__2024-11-10T00:00:00+00:00, execution_date=20241110T000000, start_date=20250124T190318, end_date=20250124T190318
[2025-01-24T19:03:18.197+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-01-24T19:03:18.205+0000] {taskinstance.py:3895} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-01-24T19:03:18.208+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
